site/sidebar.ts

```diff
@@ -13,6 +13,10 @@ export const sidebar = [
             text: 'isBase',
             link: '/is-base',
           },
+          {
```

Comment:
So for this one, as we are talking about Account Abstraction, and AA is highly connected with Wallet and Paymaster, let's have this under the Walletkit

Reply:
Updated